compiler/rustc_middle/src/ty/error.rs

```diff
@@ -286,4 +297,9 @@ impl<'tcx> TyCtxt<'tcx> {
             Err(_) => regular,
         }
     }
+
+    pub fn add_long_ty_file<G: EmissionGuarantee>(self, err: &mut Diag<'_, G>, path: PathBuf) {
+        err.note(format!("the full name for the type has been written to '{}'", path.display()));
+        err.note("consider using `--verbose` to print the full type name to the console");
+    }
```

Comment:
Question: I might be missing the obvious, but is this actually called anywhere?

Reply:
I thought I'd removed it :facepalm: 

It's a left over from before I decided to spend the time to make this follow RAII more closely, once I realized just how many uses of `short_ty_string` we had.